### DISCUSSION TITLE
Example project for adding new animations / playing animations?

### DISCUSSION THREAD

**Discussion by xkamronx** [Sixze/ALS-Refactored#278] [General]
Just starting in to a new project using Refactored, and was curious if there's any documentation or an example project for actually playing / setting up new animations? Melee, reload, etc. and how to set up their curves to actually play correctly (and replicated in multiplayer).

Thanks in advance.

**Comment by xkamronx** [comment]
Also curious; is just using Play Montage through standard means okay, or is there a better way to play a replicated montage?

**Comment by jb4twin** [comment]
Would love to do that once I got how it's working

**Comment by Joel-BB** [comment]
Any update on this? I can't figure out a reliable way to play montages.

**Comment by tornellihenrique** [comment]
This should help:
https://github.com/Sixze/ALS-Refactored/discussions/19
